### PR TITLE
KW production is actually consumption (#2362)

### PR DESCRIPTION
**Pull request by ajoga**
Hello @robertahunt. Is that PR all you expected for #2362, or am I missing something ?
Fixes https://github.com/tmrowco/electricitymap-contrib/issues/2362

### CODE REVIEW

File: parsers/KW.py [hunk 1]
```diff
@@ -11,19 +11,19 @@
...
+def fetch_consumption(zone_key='KW', session=None, logger=None):
     r = session or requests.session()
     url = 'https://www.mew.gov.kw/en'
     response = r.get(url)
     load = re.findall(r"\((\d{4,5})\)", response.text)
     load = int(load[0])
-    production = {}
-    production['unknown'] = load
+    consumption = {}
+    consumption['unknown'] = load
```
Comment (robertahunt) [inline parsers/KW.py:21]:
```suggestion
    consumption = load
```

**Comment by robertahunt** [comment]
Nice! Thanks @ajoga .

Yes, except when we have a consumption parser we don't split it by fuel type, we just give the consumption straight out, I added a suggestion to your code. Otherwise looks good!

**Comment by ajoga** [comment]
> Yes, except when we have a consumption parser we don't split it by fuel type, we just give the consumption straight out, I added a suggestion to your code. Otherwise looks good!

Thank you for the explanations, I overlooked that one. I accepted your suggestion, thanks

File: config/zones.json [hunk 2]
```diff
@@ -3069,7 +3069,7 @@
       "https://github.com/alixunderplatz"
     ],
     "parsers": {
-      "production": "KW.fetch_production"
+      "production": "KW.fetch_consumption"
```
Comment (robertahunt) [inline config/zones.json:3072]: This should also change to consumption! sorry I missed this the first time!
```suggestion
      "consumption": "KW.fetch_consumption"
```

**Comment by robertahunt** [comment]
@alixunderplatz you are correct, that would make the zone appear grey. However, I am very reluctant to save data in the database which is labelled as production but is actually consumption - I think that sets a bad precedent. I am also reluctant to show 'live' carbon intensity values for KW that are completely static because we don't have per-fuel-type information. 

That said, I can understand wanting to display the consumption profile if we have it - perhaps this is something we could add to the frontend to display the consumption profiles for information. I am also curious what other contributors think about this - it would be nice to set a standard. @systemcatch  @jarek

**Comment by jarek** [comment]
My initial thought was that 100% unknown is really pushing it.

But coming to think of it, 100% `some unknown combination of oil and gas` is not really fundamentally different from 100% `various hydro plants which have different lifetime emissions values`. And we definitely have 100% hydro areas.

With the inherent uncertainty we have on carbon intensity values, it's also not really fundamentally different from assuming that all gas plants in Singapore (which make up 95+% of generation) have the exact same intensity. Oil and gas aren't that far apart in CO2eq intensity and averaging them based on yearly statistics probably gives an error of no more than 2x. As a counterexample, it would probably be worse to have an unknown combination of gas and hydro.

I don't think it would also particularly greenwash the region (any more than the IPCC average intensity figures do), so that's also not an issue.

So in summary I think I'm fine with the region having 100% unknown-fossil if you're fine with it :+1:

**Comment by systemcatch** [comment]
@robertahunt I'm not very keen on adding consumption data to the database as production. However don't we do that exact thing already for Israel?

In this instance it's probably ok since there are no significant interconnectors to other zones and I agree with Jarek's comments on the induced errors.

**Comment by robertahunt** [comment]
Alright, sounds like there is a consensus then. Sorry @ajoga, I should have gotten more feedback before opening the issue, but then I suggest we close this, I'll make a note in the readme about this just for the future.

**Comment by corradio** [comment]
Hi all!

Thank you for providing so many insights. I've read through the different threads, and here's what I suggest:
1. Revert this PR so we keep the previous behaviour
2. Add a comment quoting "Kuwait imports power in an emergency and only for a few hours at a time" from https://github.com/tmrowco/electricitymap-contrib/pull/2457#pullrequestreview-408781556 explaining why it's acceptable to use consumption as an approximation for production. Alternatively we can keep the code for `fetch_consumption` so we keep both methods. Although they will return the same data, it will be clearer to some future contributor how to iterate on the functions.

What do you think?

**Review by corradio — DISMISSED** [review]
waiting for changes

**Review by FelixDQ — APPROVED** [review]
